lib/client/api_login_test.go

```diff
@@ -282,10 +283,27 @@ func TestTeleportClient_Login_local(t *testing.T) {
 			tc.AuthConnector = test.authConnector
 			tc.PreferOTP = test.preferOTP
 			tc.AuthenticatorAttachment = test.authenticatorAttachment
+			require.Empty(t, tc.PrivateKeyPolicy,
+				"key policy should be empty before initial login")
 
 			clock.Advance(30 * time.Second)
-			_, err = tc.Login(ctx)
+			key, err := tc.Login(ctx)
 			require.NoError(t, err)
+			require.NotNil(t, key, "no client key returned from login")
```

Comment:
Suggestion: Could we do a separate, abridged test case that covers only this?

The assertions are always the same, meaning the scenarios don't really have an influence on it.

Reply:
I'm not sure what you mean - by "this" are you referring to testing that key policy is set after the user logs in?

And after reviewing @Joerger's PR I think these test cases will need to be reworked anyway, since the policy will be saved in the profile